site/src/Consumer/EmailNotificationConsumer.php

```diff
@@ -12,26 +13,78 @@
  */
 class EmailNotificationConsumer extends AbstractNotificationConsumer
 {
+    const LIBRECORES_MAIL_ID = "abcd@gmail.com";
+
+    /**
+     * @var UserManagerInterface
+     */
+    protected $userManager;
+
+    /**
+     * @var \Swift_Mailer
+     */
+    protected $mailer;
+
+    /**
+     * @var Environment
+     */
+    protected $twig;
+
     /**
      * EmailNotificationConsumer constructor
      *
-     * @param LoggerInterface $logger
+     * @param LoggerInterface      $logger
+     * @param UserManagerInterface $userManager
+     * @param Environment    $twig
+     * @param \Swift_Mailer        $mailer
      */
-    public function __construct(LoggerInterface $logger)
+    public function __construct(LoggerInterface $logger, UserManagerInterface $userManager, Environment $twig, \Swift_Mailer $mailer)
     {
         parent::__construct($logger);
+        $this->userManager = $userManager;
+        $this->twig = $twig;
+        $this->mailer = $mailer;
+    }
+
+    protected function shouldHandle()
+    {
+        $type = $this->notification->getType();
+        // TODO: Check subscription status also from database, after merge of #387
+        switch ($type) {
+            case "site_information":
+                $this->logger->info('Processing the notification and sending an email...');
+
+                return true;
+            default:
+                $this->logger->error('Something went wrong. Please check the notification type');
+
+                return false;
+        }
     }
 
     /**
      * Handles an email Notification
      *
      * @return bool
+     *
+     * @throws \Exception (Twig)
      */
     protected function handle(): bool
     {
-        // TODO: Process the Notification($this->notification)
         echo "Email Notification Consumer";
 
+        $userEmail = $this->userManager->findUserByUsername($this->notification->getRecipient()->getUsername());
+
+        $message = (new \Swift_Message($this->notification->getSubject()))
+            ->setFrom(self::LIBRECORES_MAIL_ID)
+            ->setTo($userEmail)
+            ->setBody($this->twig->render('template/email_template.html.twig', [
```

Comment:
Always add a plain text version as well

Reply:
@agathver I'm sorry but what do we intend by doing that ?

Reply:
Emails can contain two versions, a HTML version and a plaintext version. Some people have configured their email client to always view the plaintext version. That's why it is needed. Look at the emails we send out for email address confirmation for an example how to do that.

Reply:
Swiftmailer provides an `addPart()` option to set an alternate body. Currently I've set only the notification message inside it. Do we need to add anything else?

Reply:
Please us a template also for the plaintext version so that we can modify the surrounding text. We'll need to do that to add sender and unsubscribe information.